.github/workflows/ready_for_review.yml

```diff
@@ -208,18 +224,29 @@ jobs:
           for TEAM in "${EXEMPT_TEAMS[@]}"; do
             echo "Non BE Requested Team: $TEAM"
             if echo "$PR_REQUESTED_TEAMS_JSON" | jq -e ". | index(\"$TEAM\")" > /dev/null; then
-              echo "Excemption Groups were found: $TEAM"
+              echo "Excemption Teams were found: $TEAM"
               exempt_team=true
               backend_approval_required=false
               break
+            else
+              if gh api -X GET \
+                "orgs/department-of-veterans-affairs/teams/${TEAM}/memberships/$PR_AUTHOR" \
+                >/dev/null 2>&1; then
+                echo "PR author '$PR_AUTHOR' is a member of exempt team '$TEAM'."
+                exempt_team=true
+                backend_approval_required=false
+                break
+              fi
             fi
```

Comment:
Checking if author is apart an exempt team, like identity. This would resolve the team from needing our backend approval. Sometimes there are cases when the backend approval is still being required.